File: TablaMultiplicar.java
import java.util.Scanner;

public class TablaMultiplicar {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Introduce un numero: ");
        int n = sc.nextInt();

        System.out.println("Tabla de multiplicar del " + n);
        for (int i = 1; i <= 10; i++) {
            System.out.println(n + " x " + i + " = " + (n * i));
        }

        System.out.println("Presiona ENTER para salir...");
        try {
            System.in.read();
        } catch (Exception e) {}
    }
}

File: TablaMultiplicar.c
#include <stdio.h>

int main() {
    int n, i;

    printf("Introduce un numero: ");
    scanf("%d", &n);

    printf("Tabla de multiplicar del %d\n", n);
    for (i = 1; i <= 10; i++) {
        printf("%d x %d = %d\n", n, i, n * i);
    }

    printf("Presiona ENTER para salir...");
    getchar(); getchar(); // pausa
    return 0;
}
